version/version.go

```diff
@@ -2,7 +2,7 @@ package version
 
 var (
 	// Version is the operator version
-	Version = "0.0.1"
+	Version = "4.5.0"
```

Comment:
maybe keep this at `0.0.1`. The build process should set the correct version. If we see 0.0.1 then later, we know that something went wrong with the build.

Reply:
actually, i don't know :) let's keep it 0.0.1